app/lib/frontend/dom/dom.dart

```diff
@@ -16,8 +16,7 @@ final _elementRegExp = _attributeRegExp;
 // As we want to store rawJson inside a HTML Element, it is better
 // to escape all non-trusteded characters inside it. Non-trusted
 // characters must include `</!>` characters.
-final _ldJsonEscapedCharactersRegExp =
-    RegExp(r'[^0-9a-zA-Z ,@\:\{\}\[\]\.\-"]');
+final _ldJsonEscapedCharactersRegExp = RegExp(r'[^0-9a-zA-Z ,@\.\-]');
```

Comment:
Why this change?

I get that we need to escape `"`, but why any of the others?

Reply:
Just thinking it's nice that URLs can be read even if escaping somehow fails...